src/elona/lua_env/lua_api/lua_api_skill.cpp

```diff
@@ -11,25 +11,6 @@ namespace elona
 namespace lua
 {
 
-/**
- * @luadoc
- *
- * Obtains a character's skill level in a skill or spell.
- * @tparam num skill_id the skill/spell id
- * @tparam LuaCharacter chara the character to get skill information from
- * @treturn num the character's skill/spell level
- */
-int LuaApiSkill::level(int skill_id, LuaCharacterHandle chara)
```

Comment:
The definition of `LuaApiSkill::level()` was removed, but the declaration still remains in `lua_api_skill.hpp`. It is not referred to anywhere, is it?